README.md

```diff
@@ -85,6 +85,12 @@ The region the AWS `bucket` is located in.
 
 *Default:* `us-east-1`
 
+### acl
+
+The ACL to apply to the objects.
+
+*Default:* `public-read`
```

Comment:
Cool. If you're going to make it a part of the plugin config then the
default should be moved out of there and into the index.js. Let me know
when you've updated it and I'll take a look.

Thanks for your contributions.
On Sun, 25 Oct 2015 at 20:46 Timo Zingel <notifications@github.com> wrote:

> In README.md
> <https://github.com/ember-cli-deploy/ember-cli-deploy-s3/pull/20#discussion_r42950148>
> :
>
> > @@ -85,6 +85,12 @@ The region the AWS `bucket` is located in.
> >
> >  *Default:* `us-east-1`
> >
> > +### acl
> > +
> > +The ACL to apply to the objects.
> > +
> > +*Default:* `public-read`
>
> You are right. It's not really explicit. The default already exists in
> https://github.com/ember-cli-deploy/ember-cli-deploy-s3/blob/59e083a73141bf17e67a83981ded573ff8da9caf/lib/s3.js#L68
>
> But I would prefer to move the default to another place too. I will look
> into the other defaults.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ember-cli-deploy/ember-cli-deploy-s3/pull/20/files#r42950148>
> .
>